### PR TITLE
Adding a conferences directory.

### PR DESCRIPTION
**Pull request by reid-a**
The conferences directory kicks off with a draft of an SC21 birds-of-a-feather proposal. ([rendered version](https://github.com/hpc-carpentry/coordination/blob/conferences-directory/conferences/SC21/bof_draft.md))

*TK Edited to add: call for BOF submissions* 🠒 <https://sc21.supercomputing.org/submit/birds-feather-submissions>

> A BoF submission includes:
>
> * Session leader information;
> * Selection of a topic area that best fits your BoF (see below);
> * Abstract (100-word maximum);
> * Long description (500-word maximum) clearly stating goal(s), topic, relevance to the expected HPC audience, and the expected outcome; and
> * Session format information, including a description (150-word maximum) of how you plan to organize the BoF session.

### CODE REVIEW

**Comment by tkphd** [comment]
State somewhere toward the beginning that we are aspirational carpenters, not an official Carpentry yet.

**Comment by tkphd** [comment]
"on track to be incorporated" is a forward-looking statement

**Comment by tkphd** [comment]
Abstract is 18 words over the limit.

**Comment by reid-a** [comment]
As mentioned in the July 15 coordination meeting, an important aspect of the current draft is that it's highly centered on the HPC Intro lesson, with a nod towards a successor parallel coding lesson. We have more repos than that, community input on prioritization and strategies for all the repos would be valuable. The BoF proposal should reflect this, and doesn't.

**Comment by tkphd** [comment]
Any way we can make this online-interactive?

**Comment by ocaisa** [comment]
As mentioned by @psteinb on Slack, there is a HPC Carpentry relevant workshop ([Eighth SC Workshop on Best Practices for HPC Training and Education (BPHTE21)](https://sighpceducation.acm.org/events/BPHTE21CFP.html)) at SC21 which might be another place to present HPC Carpentry (but will probably not provide a huge opportunity to discuss HPC Carpentry specific concerns). The deadline for submitting an abstract is the 20th of August.

File: conferences/SC21/bof_draft.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+It's possible to include a brief bio for these.
+
+## Topic Area
+
+ - Education
+
+## Abstract:
+
+[HPC Carpentry][1] is a community and a set of lessons for introducing new
```
Comment (ocaisa) [inline conferences/SC21/bof_draft.md:17]:
```suggestion
[HPC Carpentry][1] is a community built around a set of community-contributed lessons for introducing new
```

File: conferences/SC21/bof_draft.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+users to cluster-based high-performance computing resources. As these resources
+become more common, the need for educational resources to bring users from a
+broad variety of technical backgrounds up to speed becomes more urgent. Based
+on the pedagogically sound methods of the larger [Carpentries][2] method,
```
Comment (ocaisa) [inline conferences/SC21/bof_draft.md:21]:
```suggestion
on the pedagogically well-founded methods of the larger [Carpentries][2] approach,
```

File: conferences/SC21/bof_draft.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+need to address the gap between the needs of these users and the capabilities
+and opportunities presented by the available HPC resources.
+
+The [HPC Carpentry][1] lessons address this gap, building on the pedagogically
+sound instructional methods of the larger [Carpentries][2] community to develop
+several lessons. The best-developed of these, the [HPC Intro][3] lesson, was
+recently submitted to the [Carpentries Incubator][7] program, and is on track
+to become an official [Carpentries][2] lesson. Other important lessons in
+development include a [parallel python][4] lesson, and a lesson focusing on the
+[chapel][5] language.
```
Comment (ocaisa) [inline conferences/SC21/bof_draft.md:53]: What about the HPC Shell lesson, and I'm not sure I would highlight Chapel unless there is a serious plan to update that.

Comment (ocaisa) [reply conferences/SC21/bof_draft.md:53]: Also, in general, how do we acknowledge that other people are also creating lessons that are potentially of huge value to HPC Carpentry but are not under that umbrella (the [container lesson from the Pawsey Centre](https://pawseysc.github.io/singularity-containers/) is a good example)

File: conferences/SC21/bof_draft.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+### Goals for the BoF
+
+During this BoF, we will gather community feedback on our progress to date, to
+ensure that we are aligned with current HPC best practices, and to help the HPC
```
Comment (ocaisa) [inline conferences/SC21/bof_draft.md:58]: What does alignment mean here? Do we plan on doing some kind of live survey (via something like [Socrative](https://www.socrative.com/))? Do we put a load of options on the table for development goals?

File: conferences/SC21/bof_draft.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+## Commercial?
+
+No.
+
+## Format
+
+The BoF will open with a presentation of the current state of the lessons,
+including "pain points" identified by the development team, after which an open
+discussion will be held to solicit feedback on thees points, and the questions
```
Comment (ocaisa) [inline conferences/SC21/bof_draft.md:89]:
```suggestion
discussion will be held to solicit feedback on these points, and the questions
```

File: conferences/SC21/bof_draft.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+above, from the attendees.
+
+## Prior BoFs?
+
+An HPC Carpentry BoF was held at SC17, and an informal group of about fifteen
+participants got together at SC18.
+
+## Time, expected attendance, keywords
+
+ - We expect the session will last 2.5 to 3 hours.
```
Comment (ocaisa) [inline conferences/SC21/bof_draft.md:99]: According to the SC schedule ([here](https://sc21.supercomputing.org/submit/birds-feather-submissions/#section10)), you only get either 1 or 1.5 hours

**Review by ocaisa — COMMENTED** [review]
In general, it reads fine but I can there is scope to say exactly how we plan to efficiently gather audience feedback. The session is also actually a lot shorter than foreseen here, so there may need to be some prioritisation.

**Comment by tkphd** [comment]
At present, the Session Leader volunteers are:

- @reid-a 
- @tkphd 
- @ocaisa 
- @annajiat

File: conferences/SC21/bof_draft.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+   positive and negative aspects of the experience.
+
+    - Was the HPC Carpentry material used stand-alone, or in combination with other material?
+    - What sequence was used? 
+
+2. Our existing lessons are not uniformly developed, and we have many good ideas for future lessons. Where should we focus future lesson efforts?
+
+    - Existing lessons are fine, effort should go to improving them.
+    - HPC applications of new languages -- Julia, Chapel, Python.
+    - HPC frameworks -- Containers, Dask, MPI, MPI, parallel CUDA.
```
Comment (tkphd) [inline conferences/SC21/bof_draft.md:83]: "MPI, MPI,"

**Comment by tkphd** [comment]
@reid-a please check in the final version, then we'll merge this for posterity.

**Review by tkphd — APPROVED** [review]
Looks great!

**Pull request merged**